src/util/stat.cpp

```diff
@@ -145,7 +145,7 @@ bool Stat::track(const QString& tag,
         return false;
     }
     StatReport report;
-    report.tag = strdup(tag.toAscii().constData());
+    report.tag = strdup(tag.toLatin1().constData());
```

Comment:
toUtf8() see here for the decoding: 
https://github.com/mixxxdj/mixxx/blob/9da873a777a147a3afadfb3a988ec42112359d1e/src/util/statsmanager.cpp#L205

Reply:
Nice catch, done.